### DISCUSSION TITLE
Utils function to generate theme based on an accent color

### DISCUSSION THREAD

**Discussion by Kylar13** [shadcn-ui/ui#3755] [Ideas]
I love the way shadcn/ui does theming and their showcasing page in https://ui.shadcn.com/themes

I think it would be cool to have a way to auto-generate themes with a wider range of colors, something akin to what RadixUI does [here](https://www.radix-ui.com/colors/custom)

Are there any community projects working on this that I have missed? Knowing this project uses a lot of RadixUI stuff, could it be possible to bring the two theming solutions closer (Having RadixUI custom pallete generator export a "shadcn/ui theme" or having the shadcn theme's work with arbitrary colors

**Comment by cxa** [comment]
Check out my project here: <https://realazy.com/enough-shadcn-ui-colors/>. It might fit your needs.
